fidesctl/src/fideslang/models.py

```diff
@@ -131,10 +192,33 @@ class DataUse(FidesModel):
     legal_basis: Optional[LegalBasisEnum]
     special_category: Optional[SpecialCategoriesEnum]
     recipients: Optional[List[str]]
+    legitimate_interest: bool = False
```

Comment:
I know you already added descriptions to the docs, but can you add field descriptions here as well? This will make it possible to lean more heavily on auto-generated docs in the future

Reply:
you can look at the `FidesModel` for inspiration on how this is done

I think that should be another PR though since we might as well update all of them....I'll open one

Reply:
added a follow-on issue here: https://github.com/ethyca/fides/issues/435